tools/python/m2k-osc/analog_custom_signal.csv

```diff
@@ -0,0 +1,37 @@
+0,
```

Comment:
is this CSV file intended or committed by accident?

Reply:
this one is intended, you load the file in the tkinter app and push the values to an analog output channel. there's also a similar file to load and push data to the digital channels. they both serve as example files